addons/website_livechat/static/tests/helpers/mock_server.js

```diff
@@ -0,0 +1,38 @@
+odoo.define('website_livechat/static/tests/helpers/mock_server.js', function (require) {
+'use strict';
+
+require('im_livechat/static/tests/helpers/mock_server.js'); // ensure mail overrides are applied first
+
+const MockServer = require('web.MockServer');
+
+MockServer.include({
+    /**
+     * @override
+     */
+    _mockMailChannelChannelInfo(ids, extra_info) {
+        const channelInfos = this._super(...arguments);
+        for (const channelInfo of channelInfos) {
+            const channel = this._getRecords('mail.channel', [['id', '=', channelInfo.id]])[0];
+            // add the last message date
+            if (channel.channel_type === 'livechat') {
+                if (channelInfo.livechat_visitor_id) {
+                    const visitor = this._getRecords('website.visitor', [['id', '=', channelInfo.livechat_visitor_id]])[0];
+                    const country = this._getRecords('res.country', [['id', '=', visitor.country_id]])[0];
+                    channelInfo.visitor = {
+                        name: visitor.display_name,
```

Comment:
it's confusing to call it name and return display_name, is that what the server does?

Reply:
yes ... 
![image](https://user-images.githubusercontent.com/48204996/94110190-8c198200-fe42-11ea-9c05-6b56ee7dff94.png)

Reply:
ok...

Reply:
Let's create a task to change it in master.

Reply:
https://www.odoo.com/web#id=2345866&action=333&active_id=1750&model=project.task&view_type=form&cids=1&menu_id=4720